ansible/roles/schulcloud-server-core/templates/tldraw-server-svc.yml.j2

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: tldraw-server-svc
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: tldraw-server
+spec:
+  type: ClusterIP
+  ports:
+    # port for WebSocket connection
+    - port: 3345
+      targetPort: 3345
+      protocol: TCP
+      name: tldraw-ws
+    # port for http managing drawing data
+    - port: 3349
+      targetPort: 3349
+      protocol: TCP
+      name: tldraw-http-management
```

Comment:
Where is the referec for the port 3349 at ansible/roles/schulcloud-server-core/templates/tldraw-deployment.yml.j2 ?